Formula/b/beancount-language-server.rb

```diff
@@ -0,0 +1,19 @@
+class BeancountLanguageServer < Formula
+  desc "Language server for beancount files"
+  homepage "https://github.com/polarmutex/beancount-language-server"
+  url "https://github.com/polarmutex/beancount-language-server/archive/refs/tags/v1.3.4.tar.gz"
+  sha256 "d56f8d2d30f471b5f5011ffff9b55da9fecaec13f34a5f8f2ec0d6aa28cd3ded"
+  license "MIT"
+  head "https://github.com/polarmutex/beancount-language-server.git", branch: "main"
+
+  depends_on "rust" => :build
+  depends_on "beancount"
+
+  def install
+    system "cargo", "install", *std_cargo_args(path: "crates/lsp")
+  end
+
+  test do
+    system "#{bin}/beancount-language-server", "--help"
```

Comment:
Could you try with a test that covers some of its functionalities? Checks on version or help text almost never fail, even for broken installations.

For language servers specifically, you may want to check out [`cmake-language-server.rb`](https://github.com/Homebrew/homebrew-core/blob/6840a9fd5d0c225ca46919a0c2350932e0eba081/Formula/c/cmake-language-server.rb#L47-L60) and [`lua-language-server.rb`](https://github.com/Homebrew/homebrew-core/blob/6840a9fd5d0c225ca46919a0c2350932e0eba081/Formula/l/lua-language-server.rb#L44-L55).

Reply:
Thanks for the pointers! I've updated this to test that the server can be initialised.